monitoring/oplog-populator/alerts.yaml

```diff
@@ -0,0 +1,58 @@
+x-inputs:
+- name: namespace
+  type: constant
+  value: zenko
+- name: oplog_populator_job
+  type: config
+  value: artesca-data-backbeat-oplog-populator-headless
+- name: connectorReconfigurationFailureThreshold
+  type: config
+  value: 0.2
+- name: oplogPopulatorChangeStreamLagThreshold
+  type: config
+  value: 2
+
+groups:
+- name: Oplog Populator
+  rules:
+
+  - alert: OplogPopulatorUnavailable
+    Expr: |
+      sum(up{namespace="${namespace}",job="${oplog_populator_job}"}) < 1
+    For: "30s"
+    Labels:
+     severity: critical
+    Annotations:
+      zenko_service: backbeat-oplog-populator
+      description: "Oplog populator pod is down"
+      summary: "Oplog populator service is in critical state"
+
+  - alert: KafkaConnectFailedConnectorConfiguration
+    Expr: |
+        (sum (rate(oplog_populator_reconfiguration{success="false",job="${oplog_populator_job}",namespace="${namespace}"}[1m]))
```

Comment:
not sure the "ratio" is significant : we want the system to be "fully" configured properly, i.e. for each run of the oplog populator we expect that it configured every connector properly. If 1 connector is not configured properly, we will not get all events, which is an issue → just raise an alert.

here, this would simply mean raise an alert of `increase(oplog_populator_reconfiguration{success=false}[1m]) > 0` i guess.